README.md

```diff
@@ -87,4 +87,4 @@ If you use EDS-NLP, please cite us as below.
 
 ## Acknowledgement
 
-We would like to thank [Assistance Publique – Hôpitaux de Paris](https://www.aphp.fr/) and [AP-HP Foundation](https://fondationrechercheaphp.fr/) for funding this project.
+We would like to thank [Assistance Publique – Hôpitaux de Paris](https://www.aphp.fr/), [AP-HP Foundation](https://fondationrechercheaphp.fr/) and [INRIA](https://www.inria.fr) for funding this project.
```

Comment:
Shall we specify Inria's Soda team? https://team.inria.fr/soda/

Reply:
And it's `Inria` now apparently :open_mouth:

Reply:
> Shall we specify Inria's Soda team? https://team.inria.fr/soda/
@RomainBey ?

Reply:
My understanding is that it writes Inria and not INRIA. Last time we talked they were OK for mentioning Inria without specifying the research team.